cli/lib/kontena/cli/cloud/login_command.rb

```diff
@@ -51,6 +55,37 @@ def finish
       true
     end
 
+    def remote_login
+      client_id = kontena_account.client_id || Kontena::Client::CLIENT_ID
+      params = {
+        client_id: client_id
+      }
+      cloud_url = kontena_account.url
+      client = Kontena::Client.new(cloud_url, nil)
+      auth_request_response = client.post('/auth_requests', {}, params)
```

Comment:
i think it's supposed to be in the body (and content type `application/x-www-form-urlencoded`)